src/org/ods/orchestration/util/Project.groovy

```diff
@@ -690,14 +690,26 @@ class Project {
         envParams
     }
 
+    @NonCPS
+    private String extractClusterName(String targetApiUrl) {
```

Comment:
@s2oBCN please remove the example and consider that ODS is open source. Not all cluster API URLs will necessarily have this pattern.